scenarios/kubernetes_build.py

```diff
@@ -30,8 +30,40 @@ def check(*cmd):
     print >>sys.stderr, 'Run:', cmd
     subprocess.check_call(cmd)
 
+def check_output(*cmd):
+    """Log and run the command, raising on errors, return output"""
+    print >>sys.stderr, 'Run:', cmd
+    return subprocess.check_output(cmd)
+
+def check_build_exists(gcs):
+    """ check if a k8s/federation build with same version
+        already exists in remote path
+    """
+    version = ''
+    try:
+        check('bazel', '--batch', 'build', '//:save_git_version')
```

Comment:
a slightly better way (which is similar):
```shell
hack/print-workspace-status.sh | grep gitVersion | awk '{print $2}'
```

Reply:
Definitely use this :+1: :-)

Reply:
this is equivalent hacky-ish...

we probably should add a make target to print the version in k/k...

Reply:
This isn't very hacky, print-workspace-status.sh spits out the gitVersion (and other things) and then we're just grabbing the actual version. 

We'd have to backport a make target to older releases

Reply:
done